README.md

```diff
@@ -81,6 +82,10 @@ where `5` in the above is the number of committee members, and can be replaced b
 Running this script will result in generating log files `node0.log, node1.log, ...` corresponding to subsequent nodes.
 The script will try to start nodes at predefined IP addresses, `127.0.0.1:43XXX`, where `XXX` denotes the node id, and a node will panic if the address is not available.
 
+This example creates a directory called `aleph-bft-examples-ordering-backup` required by the crash recovery mechanism, and the logs from subsequent runs are appended to existing log files.
+Therefore, you may simulate node crashes by interrupting the `run.sh` script, and then launching it again.
```

Comment:
I don't think this is what we want to test. I know that this is named crash recovery in Jira, but the scenario was intended to test node crash recovery, not chain crash recovery. So we should kill some number of nodes below the byzantine threshold. 

Maybe you could just:
* add a separate script (named for example `run_node_crash_recovery.sh`), that would run it
* pass lower number to `--n-items` for some subset of nodes
* wait for them to finish 
* restart them
* wait for all to finish
* remove all backups once it is finished (so we don't have to remove them manually)

Alternatively you could also just kill some of the nodes, so it would simulate the crash in a more similar manner to the actual crash, but I'm not sure how easy it would be to get the right timing on that.

Also it would be nice to somehow check if blocks proposed by those killed are finalized, but I'm not sure how to do that right now.

Reply:
Strictly speaking the files are supposed to be holding the whole state, so recovering from a crash of more than the Byzantine threshold should also be supported. But yeah, w full chain crash is not the expected way this should be used.

Reply:
I've rewritten the script, now multiple scenarios can be tested